js/util/browser/dom.js

```diff
@@ -51,10 +51,19 @@ exports.suppressClick = function() {
     }, 0);
 };
 
-exports.mousePos = function (el, e) {
+var cachedOffset;
```

Comment:
This is effectively a global variable, and it makes me nervous about bugs on pages with multiple maps. Are you sure `setOffset` always gets called when necessary?